common/check_expired_locked_rules

```diff
@@ -6,51 +6,96 @@
 # You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
 #
 # Authors:
-# - Cedric Serfon, <cedric.serfon@cern.ch>, 2015
+# - Cedric Serfon, <cedric.serfon@cern.ch>, 2020. 
+# - Eric Vaandering, <ewv@fnal.gov> 2020.
+# - Fernando Garzon, <oscar.fernando.garzon.miguez@cern.ch> 2020.
 
-'''
+"""
 Probe to check the locked expired rules or datasets with locked rules
-'''
+"""
+
+from __future__ import print_function
 
 import sys
-from rucio.db.sqla.session import get_session
+from datetime import datetime
+
+from rucio.core import monitor
+from rucio.db.sqla import models
+from rucio.db.sqla.session import BASE, get_session
+from sqlalchemy import func
 
 # Exit statuses
 OK, WARNING, CRITICAL, UNKNOWN = 0, 1, 2, 3
 
+if BASE.metadata.schema:
+    schema = BASE.metadata.schema + '.'
+else:
+    schema = ''
+
 
 def main():
-    '''
+    """
     Probe to check the locked expired rules or datasets with locked rules
-    '''
+    """
     status = OK
     session = get_session()
     try:
-        query = "select rawtohex(id), scope, name, rse_expression from atlas_rucio.rules where locked=1 and expires_at<sys_extract_utc(localtimestamp)"
-        print 'Locked expired rules'
-        for row in session.execute(query):
+        print('Locked expired rules')
+        query = (session.query(models.ReplicationRule.id, models.ReplicationRule.scope, 
+        models.ReplicationRule.name, models.ReplicationRule.rse_expression))
```

Comment:
Please format that correctly. PyCharm does a nice job or just make sure the various "models" line up under each other and that the filters line up under session.

Reply:
Does it look better now?